prefq/imitation_server.py

```diff
@@ -0,0 +1,184 @@
+"""A webserver that asks the user preference queries."""
```

Comment:
Is this a leftover or why do we need a separate server?

Reply:
due to differences in implementation, several changes in have to be made. eg, the default flask server (server.py) communicates with the QueryClient (query_client.py), that can request and receive stored data from the server at any given moment, while the imitation server (imiation_server.py) is supposed to send stored feedback to the PrefqGatherer (defined in imitation_preference_comparisons.py) only when all videos have been evalutated.

Reply:
Can't we use the query client api here as well?